support/checkstyle/pom.xml

```diff
@@ -20,6 +20,10 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
+
+        <version.enforcer.plugin>3.0.0-M1</version.enforcer.plugin>
```

Comment:
That duplication isn't great, but it's required as this POM currently doesn't extend the parent one. We should change that eventually, there is an issue for it already.